tests/test_bar.py

```diff
@@ -216,3 +224,25 @@ def test_compute_fwd_and_reverse_df_over_time(frames_per_step):
     # The values at the end should be nearly identical since they contain all the samples
     assert np.allclose(fwd[-1], rev[-1])
     assert np.allclose(fwd_err[-1], rev_err[-1])
+
+
+def test_bootstrap_bar_and_regular_bar_match():
+    """In cases where the u_kln has effectively no overlap, that bootstrapping returns 0.0 as the error
+    since the BAR estimate is always zero.
```

Comment:
```suggestion
    """In cases where the u_kln has effectively no overlap, bootstrapping returns 0.0 as the error
    since the BAR estimate is always zero. Checks that `bar_with_pessimistic_uncertainty` returns the (non-zero) error from the BAR estimate computed on all samples in these cases.
```